lib/source/metadata/parser.js

```diff
@@ -116,6 +116,7 @@ Parser.mappings = {
       return;
     }
 
+    properties['iam-role'] = roles[0].substr(path.length);
```

Comment:
I'm hesitant to use `String.substr()` here where `String.replace()` is more appropriate for the operation and more obvious in its action.